app.py

```diff
@@ -1,37 +1,84 @@
+import os
 from urllib.request import urlopen
+# from urllib.error import URLError, HTTPError
 from bs4 import BeautifulSoup
+from dotenv import load_dotenv
 # import sys
+load_dotenv()
 
 
-def url_to_html(url):
-    response = urlopen(url)
-    return response.read()
+# TODO Send Push Notification
 
+KINDLE = os.getenv('KINDLE')
 
-def find_price_in_html(html):
-    content_html_page = BeautifulSoup(html, 'html.parser')
-    # price # id tag for physical books
-    # kindle-price # id tag for eBooks
-    # price_inside_buybox # id tag for physical products
 
-    return content_html_page.find(id="price").get_text()
+def url_to_html(url):
+    response = urlopen(url)
+    return response.read()
 
 
 def get_formatted_price(price):
-    price_index = price.find('$') + 2
-    price = price[price_index:len(price)+1]
+    price_start = price.find('$') + 1
+    price_end = price.find(',') + 3
+    price = price[price_start:price_end]
     price = price.replace('.', '')
     price = price.replace(',', '.')
     return float(price)
 
 
-# url = 'https://www.amazon.com.br/1984-Exclusiva-Amazon-George-Orwell/dp/6586490162/ref=tmm_hrd_swatch_0?_encoding=UTF8&qid=1631567074&sr=1-3'
+def get_price(url):
+    html = url_to_html(url)
+    content_html_page = BeautifulSoup(html, 'html.parser')
+    book_price = content_html_page.find_all(
+        'span', class_="a-size-base a-color-price a-color-price")
```

Comment:
abstrai isso em uma função

```suggestion
    book_price = get_book_price(soup_html)
```